### ISSUE TITLE
Installer and Uninstaller are falsely identified as a virus by Windows Defender

### ISSUE THREAD

**Issue by CarlKenner** [polygraphene/ALVR#141]
The mshta lines in the bat files trigger Windows Defender and give me an access denied error.

**Comment by gb2111** [comment]
probably because it is not signed. as most open source files. 
because it would require most likely pay for signature which does not make point in such case.
you should tell your AV to let this program go.

sorry, just this is just my view.

**Comment by polygraphene** [comment]
Hmm...
That is not an issue on my environment.

It may be because of Zone identifier.
Right click on the bat file and open property to check if the access is blocked by security reason.
refer https://www.thewindowsclub.com/fix-windows-blocked-access-file

**Comment by CarlKenner** [comment]
I used the installer, so they weren't marked as from the internet.

They were also blocked by Windows Defender SmartScreen, but that's a separate issue. We can't fix that. That's just because they aren't commonly run.

I did tell my AV to let it go. I know it's not a virus. But that's not the point. I want a solution that fixes it for everyone so they don't have to mess with their AV. I'll put in a pull request that solves it.

### Trojan:Win32/Powessere.G
Alert level: Severe
Date: 13/07/2018

Recommended action: Remove the threat now.

Category: Trojan
Details: This program is dangerous and executes commands from an attacker.

Affected items:
CmdLine: \Device\HarddiskVolume6\Windows\System32\mshta.exe mshta  "javascript:var sh=new ActiveXObject( 'WScript.Shell' ); sh.Popup( 'Driver Probably Installed', 10, 'Driver installer', 64 );close()"

**Comment by polygraphene** [comment]
It is strange that my Defender does not complain about that.
Anyway, I appreciate your contribution. Thank you!
